lib/public/components/Filters/RunsFilter/MagnetsFilteringModel.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { RemoteData } from '/js/src/index.js';
+import { getRemoteData } from '../../../utilities/fetch/getRemoteData.js';
+import { SelectionDropdownModel } from '../../common/selection/dropdown/SelectionDropdownModel.js';
+import { FilterModel } from '../common/FilterModel.js';
+
+/**
+ * AliceL3AndDipoleFilteringModel
+ */
+export class MagnetsFilteringModel extends FilterModel {
+    /**
+     * Constructor
+     */
+    constructor() {
+        super();
+        this._selectionDropdownModel = new SelectionDropdownModel({ multiple: false });
```

Comment:
Given that you refactor now, perhapds you can follow the standard in for example `DetectorsFilterModel` in which data is provided via a provider?